features/machine/machine.feature

```diff
@@ -308,6 +308,7 @@ Feature: Machine features testing
       | iaas_type | machineset_name        |
       | gcp       | machineset-clone-32126 | # @case_id OCP-32126
 
+    @inactive
```

Comment:
Since it is migrated, perhaps we should just remove them here, HDYT ? Currently tag inactive is used to indicate the test case is inactive in Polarion, which is not suitable for this case.